tests/api/test_encrypt_decrypt_psk.c

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
The filename `tests/api/test_encrypt_decrypt_psk.c` is too long. Let's shorten it as `tests/api/test_cipher_psk.c`

Reply:
How about changing to `test_mam_psk.c`?

Reply:
It sounds pretty well.